ext/opcache/jit/zend_jit_helpers.c

```diff
@@ -863,7 +863,7 @@ static zend_never_inline ZEND_COLD void zend_wrong_string_offset(void)
 		EMPTY_SWITCH_DEFAULT_CASE();
 	}
 	ZEND_ASSERT(msg != NULL);
-	zend_throw_error(NULL, msg);
+	zend_throw_error(NULL, "%s", msg);
```

Comment:
I think this one should be covered by a test.

Reply:
this change doesn't modify the output. why test is needed?

Reply:
It does not ... But as a fprintf like function the value needed to be treated as an argument,